### DISCUSSION TITLE
Wavelog on Home Assistant

### DISCUSSION THREAD

**Discussion by krisp** [wavelog/wavelog#1368] [Show and tell]
I am running [Home Assistant](https://www.home-assistant.io) at home using their bare metal operating system which is basically a container orchestration platform for the rest of their ecosystem. As such, I created a simple Home Assistant addon that builds Wavelog from the latest published docker container. The only prerequisite is that the MariaDB addon also be installed and configured with a database and user for Wavelog. At first run, the install wizard is presented on port 8086. The configuration is stored in the addon_config bind map and will survive uninstalls/upgrades, and the startup script will check for the config file and lock out the installer if found.

If anyone is interested, find it here on github [Wavelog Home Assistant Addon](https://github.com/krisp/homeassistant-wavelog)

**Comment by krisp** [comment]
Just wanted to bump this, it does look like folks are using it based on github taffic. I don't always update the config file as soon as a new release comes out but I do what I can. Updating to a new version is just a change to the config file so that homeassistant knows there is an update; simply running a rebuild will always grab the latest release.

**Comment by int2001** [comment]
Great that you're maintaining that HomeAssistant-Thing.
For all Users: Keep in mind: The Wavelogteam isn't able to support every distribution in detail.
If someone uses "installscripts / pakets" from other repos, the maintainer of that repo is responsible for questions/issues about it.
